homeassistant/components/arlo.py

```diff
@@ -55,4 +75,25 @@ def setup(hass, config):
             title=NOTIFICATION_TITLE,
             notification_id=NOTIFICATION_ID)
         return False
+
+    def hub_refresh(event_time):
+        """Call ArloHub to refresh information."""
+        _LOGGER.info("Updating Arlo Hub component")
+        hass.data[DATA_ARLO].hub.update(update_cameras=True,
+                                        update_base_station=True)
+        dispatcher_send(hass, SIGNAL_UPDATE_ARLO)
+
+    # register service
+    hass.services.register(DOMAIN, 'update', hub_refresh)
+
+    # register scan interval for ArloHub
+    track_time_interval(hass, hub_refresh, scan_interval)
     return True
+
+
+class ArloHub(object):
+    """Representation of the base Arlo hub component."""
+
+    def __init__(self, hub):
+        """Initialize the entity."""
+        self.hub = hub
```

Comment:
Why store the hub on another instance if it's just that attribute and no methods needed?